plugin/src/main/java/io/jenkins/plugins/casc/SecretSourceResolver.java

```diff
@@ -37,13 +37,15 @@
 
     public SecretSourceResolver(ConfigurationContext configurationContext) {
         substitutor = new StringSubstitutor(
-            StringLookupFactory.INSTANCE.interpolatorStringLookup(ImmutableMap
-                    .of("base64", Base64Lookup.INSTANCE,
-                        "fileBase64", FileBase64Lookup.INSTANCE,
-                        "readFileBase64", FileBase64Lookup.INSTANCE,
-                        "file", FileStringLookup.INSTANCE,
-                        "readFile", FileStringLookup.INSTANCE
-                    ),
+            StringLookupFactory.INSTANCE.interpolatorStringLookup(
+                ImmutableMap.<String, org.apache.commons.text.lookup.StringLookup> builder()
+                    .put("base64", Base64Lookup.INSTANCE)
+                    .put("fileBase64", FileBase64Lookup.INSTANCE)
+                    .put("readFileBase64", FileBase64Lookup.INSTANCE)
+                    .put("file", FileStringLookup.INSTANCE)
+                    .put("readFile", FileStringLookup.INSTANCE)
+                    .put("readBase64", DecodeBase64Lookup.INSTANCE)
```

Comment:
maybe we should clarify with decode?
```suggestion
                    .put("decodeBase64", DecodeBase64Lookup.INSTANCE)
                    .put("readDecodeBase64", DecodeBase64Lookup.INSTANCE)
```

Reply:
@jetersen @timja the first version was actually with `decode`. Then I saw `readFileBase64` and `readFile` and decided to follow that pattern. I am fine with using decode if this is a preference

Reply:
Maybe I am confused but the `base64` is using the encoder and not decoding.
https://github.com/oleg-nenashev/configuration-as-code-plugin/blob/0a1667f4c73d8a1ffdf83d2609a10dc573bdccb7/plugin/src/main/java/io/jenkins/plugins/casc/SecretSourceResolver.java#L166